lib/src/bit_array.dart

```diff
@@ -28,6 +28,29 @@ class BitArray extends BitSet {
     return BitArray._(data);
   }
 
+  /// Creates a bit array using a Uint8List.
+  factory BitArray.fromUint8List(Uint8List list) {
+    final data =
+        list.buffer.asUint32List(list.offsetInBytes, list.lengthInBytes >> 2);
```

Comment:
I think this will only work if the length is divisible with 4. Maybe this should throw on other cases or handle them gracefully?

Reply:
Same here, will throw for now and add handling in a future PR.